R/TealAppDriver.R

```diff
@@ -376,20 +376,22 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
 
         dots <- rlang::list2(...)
 
-        checkmate::assert_choice(dots$priority_, formals(self$set_inputs))
+        checkmate::assert_choice(dots$priority_, formals(self$set_inputs)[["priority_"]])
         self$run_js(
           sprintf(
             "Shiny.setInputValue('%s:sw.numericRange', [%f, %f], {priority: '%s'})",
             slices_input_id,
             input[[1]],
             input[[2]],
-            priority_ = dplyr::coalesce(dots$priority_, "input", .size = 1)
+            priority_ = ifelse(is.null(dots$priority_), "input", dots$priority_)
           )
         )
-        self$wait_for_idle(
-          wait = dplyr::coalesce(dots$wait_, TRUE, .size = 1),
-          timeout = dplyr::coalesce(dots$timeout_, private$timeout, .size = 1)
-        )
+        # Default `wait_` (NULL) value is TRUE.
```

Comment:
You might consider rephrasing the comment (for me it was bit confusing)  Here's a suggestion: `Invoke wait_for_idle if wait_ is TRUE or NULL,`
 or may be you can removed it completely

Reply:
You're right! It doesn't make sense unless very deep into AppDriver: eff036bb4